### PR TITLE
[WIP] Why one needs a custom service fabric transaction

### PR DESCRIPTION
**Pull request by yvesgoeleven**
addressed issue #2683

### CODE REVIEW

**Comment by yvesgoeleven** [comment]
@Particular/azure-maintainers please review

**Comment by SeanFeldman** [comment]
👍 good to go.

File: nservicebus/service-fabric/transaction-sharing.md [hunk 1]
```diff
@@ -18,6 +18,12 @@ include: saga-business-data-access
 
 snippet: ServiceFabricPersistenceSynchronizedSession-Saga
 
-Warning: Shared transaction should not be committed (`CommitAsync()`) or disposed (`Dispose()`) in the handler or the saga. When data needs to be persisted to a custom collection during handler or saga execution, a new transaction should be created using the `StateManager` property of the incoming session.
+Warning: Shared transaction should not be committed (`CommitAsync()`) or disposed (`Dispose()`) in the handler or the saga. 
+
+### Using a custom transaction
+
+When data needs to be persisted to a custom collection during handler or saga execution, a new transaction should be created using the `StateManager` property of the incoming session.
```
Comment (danielmarbach) [inline nservicebus/service-fabric/transaction-sharing.md]: Not sure this is 100% correct. Isn't the answer it depends? If you want to participate in the same transactions as sagas use the transaction prop provided, if you want tx but that doesn't need to be connected to owned tx use your own. Isn't that what we want to recommend?

Comment (SeanFeldman) [inline nservicebus/service-fabric/transaction-sharing.md]: It's a slight adjustment then. Just stating explicitly the intent.

Comment (Scooletz) [inline nservicebus/service-fabric/transaction-sharing.md]: I think that an example scenario would help a lot. Like storing some information about the current execution no matter what's the outcome of it (if it succeeds or fails). I agree with @danielmarbach that currently this paragraph does not provide this.

**Comment by weralabaj** [comment]
@Particular/azure-maintainers is there anything else to do to address Daniel's feedback? please have a look and let me know if that can be merged.

**Comment by yvesgoeleven** [comment]
@weralabaj I'll incorporate that feedback this afternoon

**Comment by yvesgoeleven** [comment]
feedback incorporated, @danielmarbach @Scooletz can you have another look?

File: nservicebus/service-fabric/transaction-sharing.md [hunk 2]
```diff
@@ -18,6 +18,12 @@ include: saga-business-data-access
 
 snippet: ServiceFabricPersistenceSynchronizedSession-Saga
 
-Warning: Shared transaction should not be committed (`CommitAsync()`) or disposed (`Dispose()`) in the handler or the saga. When data needs to be persisted to a custom collection during handler or saga execution, a new transaction should be created using the `StateManager` property of the incoming session.
+Warning: Shared transaction should not be committed (`CommitAsync()`) or disposed (`Dispose()`) in the handler or the saga. 
+
+### Using a custom transaction
+
+When data needs to be persisted to a custom collection during handler or saga execution but it should not be connected to the provided transaction, then a new transaction can be created using the `StateManager` property of the incoming session. E.g. when the information needs to be stored regardless of the outcome of the handler's execution.
```
Comment (danielmarbach) [inline nservicebus/service-fabric/transaction-sharing.md]: I think "provided" is a bit vague but let's go with this for now

**Pull request merged**